src/WebJobs.Script.WebHost/Properties/Resources.resx

```diff
@@ -259,6 +259,21 @@
     ]
 }</value>
   </data>
+  <data name="SwaggerFileDeleteError" xml:space="preserve">
+    <value>Error deleting swagger document</value>
+  </data>
+  <data name="SwaggerFileReadError" xml:space="preserve">
+    <value>Error reading swagger document</value>
+  </data>
+  <data name="SwaggerFileUpdateError" xml:space="preserve">
+    <value>Error updating swagger document</value>
+  </data>
+  <data name="SwaggerGenerateDocument" xml:space="preserve">
```

Comment:
nit: consistency, some strings have `swagger` and others `Swagger` (upper cased S), let's make them consistent.